src/main/java/net/fabricmc/mappingio/tree/MemoryMappingTree.java

```diff
@@ -839,7 +839,7 @@ public String toString() {
 		}
 
 		MethodEntry(ClassEntry owner, MethodMapping src) {
-			super(owner, src);
+			super(owner, (MemberMapping) src);
```

Comment:
Now you should be able to remove these 2 redundant casts without concerns.

Reply:
Sorry, I forget these 2. Fixed now.